package.json

```diff
@@ -22,11 +22,11 @@
     "web": "yarn workspace web start"
   },
   "dependencies": {
-    "react-native": "0.61.3"
+    "react-native": "0.61.5"
   },
   "devDependencies": {
     "@types/react": "16.9.11",
-    "@types/react-native": "0.60.22",
```

Comment:
Such a vivid imagination!  There isn't any types registered yet for react-native 0.61.5.  The lastest is 0.60.25, see: https://www.npmjs.com/package/@types/react-native

Reply:
This would trigger the console interface to choose from available type files, removing the automatic flow of installation, and worrying our users.